sdks/java/core/src/main/java/org/apache/beam/sdk/util/construction/CoderTranslators.java

```diff
@@ -220,6 +224,44 @@ public List<? extends Coder<?>> getComponents(NullableCoder<?> from) {
     };
   }
 
+  public static void overrideEncodingPositions(Schema schema) {
```

Comment:
should we instead make this a public method on SchemaTranslation?  Then the override wouldn't have to be public, it could be just done within SchemaTranslation.schemaFromProto instead of requiring a separate method call.

Reply:
Great idea Sam!